src/server.ts

```diff
@@ -38,6 +37,11 @@ export class Server {
   private configApplication(): void {
     this.app.use(bodyParser.urlencoded({ extended: true }));
     this.app.use(bodyParser.json());
+    this.app.use(morgan('tiny'));  // 'combined for more info'
+
+		/*express.static is a built i"n middleware function to serve static files.
+		We are telling express server public folder is the place to look for the static files */
+    this.app.use(express.static(path.join(__dirname, '../public')));
```

Comment:
I'm not sure what I think about it.
We shall discuss it later.

Reply:
If I remember correctly, I added these lines in the beginning of the project and then removed them. 
The removal must have been lost in one of the merge conflicts.